Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -208,6 +208,9 @@ public void GenerateBlocks(WalletSecret walletSecret)
                 if (tryToSync)
                 {
                     tryToSync = false;
+                    // TODO: This condition to have at least 3 peers is disqualifying us on testnet quite often.
```

Comment:
I think the logic is to give the node more time to connect to peers, and if none found just continue with the ones we have.

There should also be a limitation where staking is not allowed if not connected to any peer (not sure where it is)

Reply:
This has its own issue now - https://github.com/stratisproject/StratisBitcoinFullNode/issues/401
as explained there, the problem is that this delay is happening after every new block, not only once during startup